web/src/components/tasks/business-formation/review/ReviewStep.tsx

```diff
@@ -32,62 +32,86 @@ export const ReviewStep = (): ReactElement => {
-          </>
-        )}
-        <ReviewMainBusinessLocation />
+        <ReviewSection
+          buttonText={Config.formation.general.editButtonText}
+          header={"Business"}
+          stepName={"Business"}
+          testId="edit-business-name-step"
+          sectionType={"MAIN"}
+        >
+          <BusinessNameAndLegalStructure isReviewStep />
+          <ReviewBusinessSuffixAndStartDate />
+          {isForeignCCorp && (
+            <>
+              <ReviewWillPracticeLaw willPracticeLaw={state.formationFormData.willPracticeLaw} />
+              <ReviewForeignCertificate foreignGoodStandingFile={state.foreignGoodStandingFile} />
+            </>
+          )}
+          <ReviewMainBusinessLocation />
+        </ReviewSection>
         {isLP && (
           <ReviewSection
-            buttonText={Config.formation.general.editButtonText}
             header={Config.formation.fields.combinedInvestment.label}
             stepName={"Business"}
-            testId="edit-combined-investment-step"
+            sectionType={"SUBSECTION"}
           >
             <ReviewText fieldName={"combinedInvestment"} />
           </ReviewSection>
         )}
         {isLP && (
           <ReviewSection
-            buttonText={Config.formation.general.editButtonText}
             header={Config.formation.fields.withdrawals.label}
             stepName={"Business"}
-            testId="edit-withdrawls-step"
+            sectionType={"SUBSECTION"}
+            marginOverride="margin-top-4"
           >
             <ReviewText fieldName={"withdrawals"} />
           </ReviewSection>
         )}
         {isLP && <ReviewPartnership />}
         {isLP && (
           <ReviewSection
-            buttonText={Config.formation.general.editButtonText}
             header={Config.formation.fields.dissolution.label}
             stepName={"Business"}
-            testId="edit-dissolution-step"
+            sectionType={"SUBSECTION"}
+            marginOverride="margin-top-4"
           >
             <ReviewText fieldName={"dissolution"} />
           </ReviewSection>
         )}
         {hasProvisions && <ReviewProvisions />}
         {hasPurpose && (
           <ReviewSection
-            buttonText={Config.formation.general.editButtonText}
             header={Config.formation.fields.businessPurpose.label}
             stepName={"Business"}
-            testId="edit-business-purpose-step"
+            sectionType={"SUBSECTION"}
+            marginOverride="margin-top-4"
           >
             <ReviewText fieldName={"businessPurpose"} isExpandable={true} />
           </ReviewSection>
         )}
-        <ReviewRegisteredAgent />
-        {hasMembers && !isLP && <ReviewMembers />}
-        <ReviewSignatures />
-        <ReviewBillingContact />
-        <ReviewBillingServices />
+        <hr className="margin-y-205" />
+        <ReviewSection
+          buttonText={Config.formation.general.editButtonText}
+          header={"Contacts"}
```

Comment:
All these headers should not be hardcoded; they should be pulled from the same Configuration values we use to power the stepper step names